### PR TITLE
Expose chat fragment

### PR DESCRIPTION
**Pull request by sukrit-ym**

### CODE REVIEW

File: YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt [hunk 1]
```diff
@@ -0,0 +1,1098 @@
...
+    private lateinit var closeButton: ImageView
+    private lateinit var micButton: FloatingActionButton
+    private lateinit var parentLayout: View
+    private var shouldKeepApplicationInBackground = true
+    private var isAgentConnected = false
+    private var hasAudioPermissionInManifest = false
+    private val TAG = "YMChat"
+    private lateinit var myWebView: WebView
+
+    // private lateinit var myWebView: WebView
```
Comment (PurushYM) [inline YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt:66]: you can remove them if not needed

File: YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt [hunk 2]
```diff
@@ -0,0 +1,1098 @@
...
+                        setMultiFileUpload(false)
+                    }
+                } catch (e: java.lang.Exception) {
+                    //Exception Occurred
+                }
+            }
+        }
+
+
+        /*ViewCompat.setOnApplyWindowInsetsListener(
```
Comment (PurushYM) [inline YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt:231]: +1

File: YMChat/src/main/java/com/yellowmessenger/ymchat/models/JavaScriptInterface.java [hunk 3]
```diff
@@ -40,12 +39,6 @@ public void receiveMessage(String s) {
             incomingEvent = new YMBotEventResponse(null, null, false);
         }
 
-        if (incomingEvent.getCode() != null && incomingEvent.getCode().equals("start-mic")) {
```
Comment (PurushYM) [inline YMChat/src/main/java/com/yellowmessenger/ymchat/models/JavaScriptInterface.java:43]: Please check that it doesn't affect any customer

**Review by PurushYM — APPROVED** [review]
Check comment once and merge the PR

File: YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt (hunk 1 above)
Comment (sukrit-ym) [reply YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt]: Done

File: YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt (hunk 2 above)
Comment (sukrit-ym) [reply YMChat/src/main/java/com/yellowmessenger/ymchat/YellowBotWebviewFragment.kt]: Done

File: YMChat/src/main/java/com/yellowmessenger/ymchat/models/JavaScriptInterface.java (hunk 3 above)
Comment (sukrit-ym) [reply YMChat/src/main/java/com/yellowmessenger/ymchat/models/JavaScriptInterface.java:43]: No impact as the event was not handled

**Pull request merged**